client/my-sites/themes/theme-options.js

```diff
@@ -206,6 +206,7 @@ function getAllThemeOptions( { translate, isFSEActive } ) {
 		getUrl: ( state, themeId, siteId, options ) =>
 			addQueryArgs( getCustomizeUrl( state, themeId, siteId, isFSEActive ), {
 				style_variation: options?.styleVariationSlug,
+				from: 'theme-browser',
```

Comment:
```suggestion
				from: 'theme-info',
```

Nitpick: I think this is referred to as the "info" screen -- it would be good to match the same wording.

<img width="453" alt="Screen Shot 2023-09-27 at 1 48 48 pm" src="https://github.com/Automattic/wp-calypso/assets/9312929/4af84e58-1954-4be5-bc40-9dc6fea98e8e">

Reply:
Good catch!